backend/src/models/preQuestionsList.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "question": "What topic(s) would you like to discuss?",
+    "type": "bullet",
+    "id": ""
+  },
```

Comment:
In `postQuestionsList.json`, the `id` fields are numbers (all 0 actually), but here they're strings. Are they intended to be different data types? If not, then for consistency I'd make them the same type

Reply:
^^ we should remove id